FileAPI/idlharness.html

```diff
@@ -27,10 +27,7 @@ <h1>File API automated IDL tests</h1>
         ['dom', 'html', 'url'],
         idl_array => {
           idl_array.add_objects({
-            Blob: ['new Blob(["TEST"])'],
-            File: ['new File(["myFileBits"], "myFileName")'],
             FileList: ['document.querySelector("#fileChooser").files'],
```

Comment:
Nit: Inconsistent use of trailing comma.

Should be harmless with modern JS, but consistency is nice. :)